docs/semgrep-app/getting-started-with-semgrep-app.md

```diff
@@ -0,0 +1,285 @@
+
+**Path ignores**
+	Paths and files specified here are not scanned by Semgrep App.
+
+To see additional scan parameters:
+
+1. Click **Projects **on the left sidebar.
+2. Select the name of the project to modify.
+3. Make edits as necessary.
+
+
+### Adding rules and rulesets to scan with
+
+Semgrep App's Rule Board displays all rules and rulesets that are used to scan repositories. These rules are scanned based on the repository's programming language and framework as well as additional Semgrep parameters, such as ignored files.
+
+For example, given five repositories each with different programming languages, the Rule Board only scans using rules and rulesets for that repository's language that are in the Rule Board.
+
+Semgrep's speed is not affected by having multiple rules for different languages in the Rule Board.
+
+You may select rules and rulesets from your own rules, your organization's rules, or rules from the Registry.
+
+
+
+![alt_text](../img/rule-board.png "image_tooltip")
+
+
+The Rule Board is composed of three columns:
+
+
+<dl>
+    <dt>Audit</dt>
+    <dd>Rules here show findings only on Semgrep App.</dd>
+    <dt>PR/MR Comments</dt>
+    <dd>Rules here show findings to developers through PRs or MRs.</dd>
+    <dt>Block</dt>
+    <dd>Rules here show block merges and commits, in addition to showing findings in Semgrep App and PRs or MRs.</dd>
+</dl>    
+
+To add rules and rulesets to your Rule Board:
+
+1. Click **Rule Board** on the left sidebar.
+2. Click Add Rules. A right-side drawer appears.
+3. Type in a search term relevant to your codebase's framework or programming language.
+4. Drag a card from the search results to the appropriate column.
+5. Select **Save changes**.
+
+For more information on operations such as filtering and deleting as well as Rule board management, see [Rule board](https://semgrep.dev/docs/semgrep-app/rule-board/).
+
+
+## Viewing and managing findings
+
+
+### Viewing findings of a scan
+
+
+![alt_text](../img/dashboard-view.png "image_tooltip")
+
+
+Both the Dashboard and the Findings page display the results of a scan. These pages are accessible from the left sidebar in Semgrep App. The **[Dashboard](https://semgrep.dev/docs/semgrep-app/dashboard/)** is a report view to assist in evaluating security posture across repositories. It organizes findings into OWASP categories, enabling users to assess habits and trends within their team or organization.
+
+The **[Findings](https://semgrep.dev/docs/semgrep-app/findings/#managing-triage-states-bulk-triage)** page is used to triage findings. Triaging refers to prioritizing a finding based on criteria set by your team or organization. While severity is a factor in triage, your organization may define additional criteria based on coding standards, business, or product goals.
+
+To see the rule specifics that triggered the finding, click on the rule entry.
+
+
+### Automatically resolving findings
+
+

```

Comment:
@s-santillan we have a lot of additional empty lines in this whole document, if you want, you can delete them... just to make the source code cleaner.

Generally, I think that Markdown builders ignore more than one empty line, so there is no difference after build. It just does not look that clean, but it is the tinniest thing here.

Reply:
I'll pare down some extra whitespace